pkg/mons/restore_quorum.go

```diff
@@ -304,6 +310,31 @@ func getMonDetails(goodMon string, monEndpoints []string) ([]string, string, str
 	return badMons, goodMonPublicIp, goodMonPort, nil
 }
 
+func validateMonIsUp(ctx context.Context, clientsets *k8sutil.Clientsets, clusterNamespace, monID string) error {
+	args := []string{"daemon", fmt.Sprintf("mon.%s", monID), "mon_status"}
+	out, err := exec.RunCommandInLabeledPod(ctx, clientsets, fmt.Sprintf("mon=%s", monID), "mon", "ceph", args, clusterNamespace, true)
+	if err != nil {
+		return fmt.Errorf("failed to run command ceph daemon mon.%s mon_status %v", monID, err)
+	}
+
+	type monStatus struct {
+		State string `json:"state"`
+	}
+	var monStatusOut monStatus
+	err = json.Unmarshal([]byte(out), &monStatusOut)
+	if err != nil {
+		return fmt.Errorf("failed to unmarshal mon stat output")
+	}
+
+	logging.Info("mon %s state is %s", monID, monStatusOut.State)
```

Comment:
The logs are easier to read vars when quotes around them
```suggestion
	logging.Info("mon %q state is %q", monID, monStatusOut.State)
```

Reply:
done